### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by yash01234**
Updating the markdown table with latest variable names

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,3 +86,17 @@ if(score >=60 && score <= 64){
 if(score <= 59){
   console.log("Grade: Redo");
 }
+
+var table = require('markdown-table')
+  
+var t= table([
```
Comment (dangoslen) [inline index.js:100]: Very cool. I'm glad there was a dep out there for this.

However, I don't see it in the `package.json`. How did you install it? We need to make sure it's added as a dependency in that file

**Comment by dangoslen** [comment]
I think the changes look good, but the black box test isn't running - https://github.com/dangoslen/constable-github-action/pull/35/checks?check_run_id=1130752918

Not sure, but it looks like something is happening when trying to generate the table

**Comment by yash01234** [comment]
how do we go about fixing this?

**Comment by tirthpatel7498** [comment]
@yash01234 Run `npm run package` command after you update the index.js file. That will make changes to the dist/index.js file as well coz there's a difference in those 2 files.

**Comment by tirthpatel7498** [comment]
Also, can you take `index.js` again from main branch and copy your code for the table in that. I think you are using the old `index.js` file. May be the checks are not passing because of that.

File: index.js [hunk 2]
```diff
@@ -86,3 +95,16 @@ if(score >=60 && score <= 64){
 if(score <= 59){
   console.log("Grade: Redo");
 }
+var table = require('markdown-table')
+  
+var t= table([
+  ['Item', 'Weight', 'Score'],
+  ['README.md','2', readme_present],
+  ['CONTRIBUTING.md','1', contributions_present]
```
Comment (dangoslen) [inline index.js]: You are missing `,` after each row

**Comment by dangoslen** [comment]
@yash01234 you need to update the `dist/index.js` file using the `npm run package` command. Once you do that, just commit and push those changes and it should be good.

**Comment by yash01234** [comment]
When I run (npm run package) command I get an error. I go onto check the logs and I see- (error missing script: markdown-table) as an error for some reason

**Comment by dangoslen** [comment]
interesting. sounds like you have an odd environment that isn't including everything in your node modules directory. 

have you done `npm install` since the `markdown-table` was added to the `package.json`?

**Comment by yash01234** [comment]
Yes works now. Thanks :)

**Comment by dangoslen** [comment]
awesome! feel free to merge

**Comment by yash01234** [comment]
Done

**Pull request merged**